### PR TITLE
refs #37 - Workers now use group consumer

### PR DESCRIPTION
**Pull request by jrmbrgs**

### CODE REVIEW

File: Library/Tasks/AbstractTask.php [hunk 1]
```diff
@@ -113,14 +114,16 @@ private function parseOpt(array $paramList)
         $this->lock();
         $this->initWorker();
 
-        $this->kafkaTopicConsumer = $this->kafkaConsumer->newTopic($this->topicName, $this->kafkaTopicConf);
-        $this->kafkaTopicConsumer->consumeStart($this->topicPartitionNo, RD_KAFKA_OFFSET_STORED);
         // xxx Factorize stdout/err support
-        $this->getDI()->get('logger')->info("Worker listening on \033[32m" .
+        $this->getDI()->get('logger')->info(
+            "Worker listening on \033[32m" .
```
Comment (adefretin) [inline Library/Tasks/AbstractTask.php:119]: perhaps we could make a func dedicated to display colored logger's msg

File: Library/Tasks/AbstractTask.php [hunk 2]
```diff
@@ -113,14 +114,16 @@ private function parseOpt(array $paramList)
...
             implode(',', $this->workflowConfig['brokerServerList']->toArray()) .
-            ":\033[32m" . $this->topicName . "\033[0m");
+            ":\033[32m" . $this->topicName . "\033[0m"
+        );
+        $this->kafkaConsumer->subscribe([$this->topicName]);
         while (true) {
-            $msg = $this->kafkaTopicConsumer->consume($this->topicPartitionNo, 100);
+            sleep(1);
```
Comment (adefretin) [inline Library/Tasks/AbstractTask.php:125]: why ?

File: Library/Tasks/AbstractTask.php [hunk 3]
```diff
@@ -113,14 +114,16 @@ private function parseOpt(array $paramList)
...
+            $msg = $this->kafkaConsumer->consume(10000);
```
Comment (adefretin) [inline Library/Tasks/AbstractTask.php:126]: 10000 ?

File: Library/Tasks/AbstractTask.php (hunk 1 above)
Comment (jrmbrgs) [reply Library/Tasks/AbstractTask.php:119]: right, in another PR 👍

File: Library/Tasks/AbstractTask.php (hunk 3 above)
Comment (jrmbrgs) [reply Library/Tasks/AbstractTask.php:126]: right this is a timeout, still have to deal w/

File: Library/Tasks/AbstractTask.php (hunk 2 above)
Comment (jrmbrgs) [reply Library/Tasks/AbstractTask.php:125]: because we are not that hurry ;)

**Pull request merged**